### ISSUE TITLE
Tool report aggregation and prioritisation project 

### ISSUE THREAD

**Issue by bgruening** [usegalaxy-au/elixir-biocommons-colab#18]
The motivation is to pull together tool error reports and get a better overview of how we spent our tool-fixing-capacity effectively.

**Comment by bgruening** [comment]
@neoformit can you maybe add a summary of our discussion here? What needs to be done and if/how we can help?

**Comment by neoformit** [comment]
We would like to send tool error reports to Sentry (the galaxyproject server). To make these reports more useful for identifying problematic tools we would like to update the `sentry.py` tool error plugin to include a measure of total tool runs (probably run count in last 30 days).

Sentry seems to be capable of most of the features we require, such as clustering and enumeration of error messages and version tracking.

**Comment by bgruening** [comment]
@neoformit do you have access to sentry? Does this idea work for you, can we close this issue?

**Comment by neoformit** [comment]
Yep, I think it's good. I haven't had time to work on it yet unfortunately.

**Comment by bgruening** [comment]
Some improvements are here: https://github.com/galaxyproject/galaxy/pull/16686
